.github/workflows/build.yml

```diff
@@ -31,6 +31,13 @@ jobs:
       exec-repo: ${{ steps.ghcr-repo.outputs.exec }}
       exec-meta: ${{ steps.meta-exec.outputs.json }}
 
+    # https://docs.docker.com/build/ci/github-actions/local-registry/
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
+
```

Comment:
Our builds when we change the baseimage within the PR were broken. We don't change the baseimage very often, and it looks like a change to docker buildx means that images are built within a container now, so you can't load a local image. You have to run a local registry and load the image from that registry. Those changes could be a separate PR, but since they are hard to test without a base image, I'm including them here with the poetry changes, since they change the base image and test this.